packages/utils/src/provider/jsonRpcHttpClient.ts

```diff
@@ -43,15 +43,6 @@ export type ReqOpts = {
   shouldRetry?: (lastError: Error) => boolean;
 };
 
-export type JsonRpcHttpClientMetrics = {
```

Comment:
Why remove from here?

Reply:
@dapplion this got moved to `/packages/engine-api-client/src/provider/metrics.ts` so as to be part of the `engine-api-client` package. Will you have this be somewhere else instead?